### DISCUSSION TITLE
Set parameters

### DISCUSSION THREAD

**Discussion by jorisvanherp** [egnerfl/alfen_wallbox#14] [Q&A]
Hi,

Would it be possible to add a service to update different parameters?
The following would be needed for my use case:
- Solar charging mode
- Maximum current for load balancing
- Maximum safe current

If anyone could create this, or guide me in how to do this, that would be great! I'm sure others have the same requirements.

Joris

**Comment by leeyuentuen** [comment]
maybe take a look at the new fork version?
https://github.com/leeyuentuen/alfen_wallbox
